modules/react/skeleton/lib/parts/skeletonHeader.tsx

```diff
@@ -1,5 +1,8 @@
-import canvas from '@workday/canvas-kit-react/tokens';
 import * as React from 'react';
+
+import {createComponent} from '@workday/canvas-kit-react/common';
+import canvas from '@workday/canvas-kit-react/tokens';
+
 import SkeletonShape from './skeletonShape';
 import {TEXT_BORDER_RADIUS} from './utils';
 
```

Comment:
Outside the change area, but `SkeletonHeaderProps` no longer needs to extend from `React.HTMLAttributes<HTMLDivElement>` anymore. `createComponent` takes care of the interface. These attributes actually get stripped out and reapplied. This only increases TS processing. You'll have to manually add in `children` and `backgroundRadius` though.

Reply:
FYI, `children` and `borderRadius` were not props to this component. Cleaned up the interface though.